.github/workflows/eslint.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
             echo "ESLint version: "`npx eslint --version`
             echo "The files will be checked: "`echo ${changed_files_eslint}`
-            npx eslint ${changed_files_eslint} -f node_modules/eslint-detailed-reporter/lib/detailed.js -o ./eslint_report/eslint_checks.html
+            npx eslint --no-error-on-unmatched-pattern ${changed_files_eslint} -f node_modules/eslint-detailed-reporter/lib/detailed.js -o ./eslint_report/eslint_checks.html
```

Comment:
What are you trying to solve?

Reply:
Original issue:

ESLint version: v7.11.0

The files will be checked: cvat-core/src/annotations-collection.js cvat-core/src/annotations-filter.js cvat-core/src/annotations.js cvat-core/src/session.js cvat-core/tests/internal/filter.js cvat-ui/src/actions/annotation-actions.ts cvat-ui/src/components/annotation-page/annotation-page.tsx **cvat-ui/src/components/annotation-page/annotations-filters-input.tsx** cvat-ui/src/components/annotation-page/attribute-annotation-workspace/attribute-annotation-sidebar/attribute-annotation-sidebar.tsx cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-list-header.tsx cvat-ui/src/components/annotation-page/top-bar/filters-modal.tsx cvat-ui/src/components/annotation-page/top-bar/right-group.tsx cvat-ui/src/components/annotation-page/top-bar/top-bar.tsx cvat-ui/src/containers/annotation-page/standard-workspace/objects-side-bar/objects-list.tsx cvat-ui/src/containers/annotation-page/top-bar/filters-modal.tsx cvat-ui/src/containers/annotation-page/top-bar/top-bar.tsx cvat-ui/src/icons.tsx cvat-ui/src/reducers/annotation-reducer.ts cvat-ui/src/reducers/interfaces.ts tests/cypress/integration/actions_tasks_objects/case_18_filters_functionality.js

Oops! Something went wrong! :(

ESLint: 7.11.0

No files matching the pattern **"cvat-ui/src/components/annotation-page/annotations-filters-input.tsx"** were found.

Please check for typing mistakes in the pattern.

Error: Process completed with exit code 2.

This file **cvat-ui/src/components/annotation-page/annotations-filters-input.tsx** was deleted, but condition collect all files in PR.

Reply:
Got it. Was not it fixed in #2888?

Reply:
Maybe, but not merged to develop for sure. Any way it ill be inlined in next develop merge I think

Reply:
@vnishukov , it was merged into develop. Probably we can remove the line.